data/hai ships.txt

```diff
@@ -45,7 +45,7 @@ ship "Aphid"
 	explode "tiny explosion" 10
 	explode "small explosion" 20
 	explode "medium explosion" 15
-	description "The Aphid is the lightest freighter used by the Hai and was for many millennia the backbone of the Hai inter-planetary economy. Given that a captain does not have to fear being attacked in much of Hai space, Aphids are only lightly armed and mainly rely on larger Hai ships for protection."
+	description "The Aphid is the lightest freighter used by the Hai and was for many millennia the backbone of the Hai inter-planetary economy. Because of Hai space being peaceful for much of history, Aphids are only lightly armed and mainly rely on larger Hai ships for protection from piracy."
```

Comment:
"its recent history" would sound better

Reply:
3/4ths of the 120,000 years of Hai history has been spent in the space that they have now, and from what we can tell, only recently has it not been peaceful (Korath Raids and Unfettered aggression).

Reply:
Okay was suggesting since it sounded weird

Reply:
without "its", it sounds like history may imply any history even from when the universe started to exist
it's not a big deal but if ya add "its" it's more clear that it's about Hai space (even though it already kinda is)

Reply:
**History**: History is the study of the past as it is described in written documents. Events occurring before written record are considered **prehistory**.

Abstracting the written documents part to simply recorded information, saying just `history` when speaking in relation to the Hai would 1) already imply Hai history and 2) not necessarily imply all of happening since the Big Bang.

Reply:
Try this:
`Because of how peaceful Hai space has been for much of their history, [...]`

Also, I don't agree with the addition about piracy. The previous one more generally refers to any possible threat to the Hai.

Reply:
"from piracy" seems two narrow.